apps/keyboard/test/unit/keyboard/layout_normalizer_test.js

```diff
@@ -24,7 +24,6 @@ var range = (from, to) => {
 
 suite('LayoutKeyNormalizer', function() {
   var SPECIAL_CODES_MAP = {
-    'VK_BACK_SPACE': KeyEvent.DOM_VK_BACK_SPACE,
```

Comment:
If the key is removed from being a special key, tests which assert whether the key is a special key will undoubtedly fail. If the previous comment follows to manage backspace as a special case in js/view/key_view.js then this might need be reverted.